app/marketing/models.py

```diff
@@ -420,7 +420,7 @@ class UpcomingDate(SuperModel):
     """Define the upcoming date model"""
     # These fields are meant to use for update UpcomingDate based on the icalendar updates
     uid = models.CharField(max_length=255, null=True, blank=True)
-    last_modified = models.DateTimeField(db_index=True, auto_now=True)
+    last_modified = models.DateTimeField(db_index=True)
```

Comment:
is there a reason for this? to the best of my knowledge, removing this means this field would no longer be updated when a record is changed.

If you're doing this because of failing tests, what you want to do is a `unittest.mock`

Reply:
This field needs to be filled manually from an input iCalendar file. I had added a ```auto_now=True``` to it accidentally. So yes definitely this attribute need to be deleted in my opinion.

Reply:
This field is different from ```modified_on``` that is defined in SuperModel and tracks last modification time of the object.

Reply:
since we already have modified_on then we dont need those one as well

Reply:
As may you already knew, this field is related to the last_modified field of an actual event modification date in an iCal file @danlipert. I used this field to check if some event needs to get updated. 

If I remove this field and use modified_on then there will be no modification date of actual db record, Is that ok?